be/test/env/s3_client_test.cpp

```diff
@@ -0,0 +1,103 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
Move this file under the directory be/test/object_store

Reply:
Yes. Will Fix that.